packages/contracts/src/utils/core_combinatorial_utils.ts

```diff
@@ -193,16 +275,16 @@ export class CoreCombinatorialUtils {
                 },
                 takerAssetFillAmountScenario: fillScenarioArray[9] as TakerAssetFillAmountScenario,
                 makerStateScenario: {
-                    traderAssetBalance: BalanceAmountScenario.Higher,
-                    traderAssetAllowance: AllowanceAmountScenario.Higher,
-                    zrxFeeBalance: BalanceAmountScenario.Higher,
-                    zrxFeeAllowance: AllowanceAmountScenario.Higher,
+                    traderAssetBalance: fillScenarioArray[10] as BalanceAmountScenario,
```

Comment:
What does `no-magic-numbers` have to say about it?

Reply:
Ah, it seems that `no-magic-numbers` is disables for `contracts` package... We should enable it in a separate PR.

Reply:
Agree. But I still think that we shouldn't have magic numbers in this PR. Why do we have that as an array? Why not object with properties?

Reply:
Agreed, will add a TSLint-disable. The `_getAllCombinations` method is used to be agnostic to the scenarios passed into it. This makes it more easily reusable (Alex is going to re-use it for combinatorially testing inner-methods).